charts/wbaas-backup/templates/_helpers.tpl

```diff
@@ -28,8 +28,8 @@ lifecycle:
 {{- end }}
 resources:
   limits:
-    cpu: 750m
```

Comment:
I'm pretty sure the CPU is well over this new limit, did you try this out somewhere?

Reply:
![image](https://user-images.githubusercontent.com/352149/166439243-cd62b14f-0fc1-4d34-bc8b-df1a5dff37dd.png)

from staging

Reply:
I just looked at these graphs and saw that the `used` value was way lower than the request most of the time, but your screenshot tells another story. Maybe I was looking at the wrong graph?
![image](https://user-images.githubusercontent.com/91744937/166652353-612484d0-bb6f-464b-8547-da00f1347349.png)